From 02d470e0ca5d880026dea4f9b49c0f5e9ad88cd0 Mon Sep 17 00:00:00 2001
From: Marcello Sylvester Bauer <marcello.bauer@9elements.com>
Date: Tue, 3 Feb 2026 16:10:37 +0100
Subject: [PATCH] ARM: dts: hpe-gxp: add CHIF node

Add device tree node for the GXP Channel Interface (CHIF) used for
host-BMC communication via shared memory.

Upstream-Status: Pending

Signed-off-by: Marcello Sylvester Bauer <marcello.bauer@9elements.com>
---
 arch/arm/boot/dts/hpe/hpe-gxp.dts | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm/boot/dts/hpe/hpe-gxp.dts b/arch/arm/boot/dts/hpe/hpe-gxp.dts
index 7b77be541d66..8a4d7f00ea6f 100644
--- a/arch/arm/boot/dts/hpe/hpe-gxp.dts
+++ b/arch/arm/boot/dts/hpe/hpe-gxp.dts
@@ -213,6 +213,12 @@ fn2: fn2@80200000 {
 				"post-complete",
 				/* 4: Button state (input) */
 				"btn-state";
+
+			chif: chif {
+				compatible = "hpe,gxp-chif";
+				interrupts = <12>;
+				interrupt-parent = <&vic1>;
+			};
 		};
 
 		espi: espi@80fe0000 {
